ClassIsland/App.Services.xaml.cs

```diff
@@ -244,7 +244,9 @@ private void ConfigureServices(HostBuilderContext context, IServiceCollection se
         // 认证提供方
         services.AddAuthorizeProvider<PasswordAuthorizeProvider>();
         // 语音提供方
+#if Platforms_Windows
         services.AddSpeechProvider<SystemSpeechService>();
+#endif
```

Comment:
这里应该使用 OperatingSystem.IsWindows() 判断，在 ClassIsland 项目里读不到这个常量


```suggestion
if (System.OperatingSystem.IsWindows()) {
    services.AddSpeechProvider<SystemSpeechService>();
}
```